nuke/cryptomatte_utilities.py

```diff
@@ -350,6 +355,44 @@ def test_manifest(self, quiet=False):
 
         return errors, collisions
 
+    def setup_wwfx_arnold_layers(self):
+        """Parse and update Arnold style `crypto_*` layers if WWFX in name.
```

Comment:
RST style monospace:

```suggestion
        """Parse and update Arnold style ``crypto_*`` layers if WWFX in name.
```